slacker/__init__.py

```diff
@@ -378,12 +378,13 @@ def post_message(self, channel, text=None, username=None, as_user=None,
                              'icon_url': icon_url,
                              'icon_emoji': icon_emoji,
                              'thread_ts': thread_ts,
-                             'reply_broadcast': reply_broadcast
+                             'reply_broadcast': reply_broadcast,
+                             'blocks': blocks
                          })
 
-    def me_message(self, channel, text):
+    def me_message(self, channel, text, blocks):
```

Comment:
This argument is not mentioned in the [documentation](https://api.slack.com/methods/chat.meMessage). Are you sure that it's supported? This need to be an optional argument regardless. Otherwise, it's not going to be backwards compatible.

Reply:
Can you also clean merge your commits into one with a [good commit message](https://chris.beams.io/posts/git-commit/)?

Reply:
I'll clean up the commits. Your call on me_message is correct, it doesn't (seem) to support blocks. Withdrawing this PR, will submit another clean one.

Reply:
I'm straight up too new and dumb to figure out how to rebase and squash commits in a way that is worth it. I am *certainly* not offended if you manually want to add Blocks in the like, 8 places it goes before I figure this out.